src/pudl/metadata/resources/ferc714.py

```diff
@@ -59,6 +59,47 @@
         "etl_group": "ferc714",
         "create_database_schema": False,
     },
+    "out_ferc714__hourly_planning_area_demand": {
+        "description": (
+            "Hourly electricity demand by planning area. FERC Form 714, Part III, "
+            "Schedule 2a. This table includes data from the pre-2021 CSV raw source "
+            "as well as the newer 2021 through present XBRL raw source.\n\nAn important "
+            "caveat to note is that there was some cleaning done to the datetime_utc "
+            "timestamps. The Form 714 includes sparse documentation for respondents "
+            "for how to interpret timestamps - the form asks respondents to provide "
+            "24 instances of hourly demand for each day. The form is labeled with hour "
+            "1-24. There is no indication if hour 1 begins at midnight.\n\nThe XBRL data "
+            "contained several formats of timestamps. Most records corresponding to hour "
+            "1 of the Form have a timestamp with hour 1 as T1. About two thirds of the records "
+            "in the hour 24 location of the form have a timestamp with an hour reported as "
+            "T24 while the remaining third report this as T00 of the next day. T24 is not a "
+            "valid format for the hour of a datetime, so we convert these T24 hours into "
+            "T00 of the next day. A smaller subset of the respondents reports the 24th hour "
+            "as the last second of the day - we also convert these records to the T00 of the "
+            "next day.\n\nThis table includes three respondent ID columns: one from the "
+            "CSV raw source, one from the XBRL raw source and another that is PUDL-derived "
+            "that links those two source ID's together. This table has filled in source IDs "
+            "for all records so you can select the full timeseries for a given respondent from "
+            "any of these three IDs."
```

Comment:
Rather than cut and pasting the entire description here, it might make sense to reference the core table with a minimal explanation of where it's coming from, and then describe what is new about this table compared to that one in more detail, i.e. the imputed values, which aren't mentioned here.